src/libraries/System.Runtime.Loader/tests/ApplyUpdateUtil.cs

```diff
@@ -133,5 +133,26 @@ internal static void AddRemoteInvokeOptions (ref RemoteInvokeOptions options)
                 testBody(arg1);
             }
         }
+
+
+        public static void ClearAllReflectionCaches()
```

Comment:
It would be nice someday to use the more general update handler logic from the SDK [here](https://github.com/dotnet/sdk/blob/96f5c8237d25204f9879f55ce4169b76deb18d26/src/BuiltInTools/DotNetDeltaApplier/HotReloadAgent.cs#L13) somehow.  It should work for mono too.